### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SyamBabu-M**

### CODE REVIEW

File: src/pages/Dashboard/Settings/WidgetSettings/WidgetSettings.jsx [hunk 1]
```diff
@@ -152,26 +152,41 @@ const WidgetSettings = () => {
       const searchEventsFilters = encodeURIComponent(filtersParam);
 
       const locale = onLanguageSelect(allValues?.language);
-      const urlCopy = new URL('https://s3.ca-central-1.amazonaws.com/staging.cms-widget.footlight.io/index.html');
+      const urlCopy = new URL(widgetUrl);
+      const urlCopyMobile = new URL(widgetUrl);
```
Comment (sahalali) [inline src/pages/Dashboard/Settings/WidgetSettings/WidgetSettings.jsx:156]: We don't need to define urlCopy and urlCopyMobile using new URL. Instead, we can directly assign urlCopy to urlCopyMobile.

File: src/pages/Dashboard/Settings/WidgetSettings/WidgetSettings.jsx [hunk 2]
```diff
@@ -303,7 +318,9 @@ const WidgetSettings = () => {
   }, [initialEntitiesOrganization]);
 
   useEffect(() => {
-    const urlCopy = new URL('https://s3.ca-central-1.amazonaws.com/staging.cms-widget.footlight.io/index.html');
+    const urlCopy = new URL(widgetUrl);
+    const urlCopyMobile = new URL(widgetUrl);
```
Comment (sahalali) [inline src/pages/Dashboard/Settings/WidgetSettings/WidgetSettings.jsx:322]: Similarly here

Comment (SyamBabu-M) [reply src/pages/Dashboard/Settings/WidgetSettings/WidgetSettings.jsx:322]: there are 2 widgets embedded in that page and the URL used in mobile sized widget does not respond to height parameter(the other one responds to the change in height). That is the reason 2 URLs are used here.

File: src/pages/Dashboard/Settings/WidgetSettings/WidgetSettings.jsx (hunk 1 above)
Comment (SyamBabu-M) [reply src/pages/Dashboard/Settings/WidgetSettings/WidgetSettings.jsx:156]: if we use 
const urlCopy = new URL(widgetUrl);
const urlCopyMobile = urlCopy;

to define them, then each change made to urlCopy will be applied on urlCopyMobile as they share the same reference.

We need both of them to be independant.

**Review by sahalali — APPROVED** [review]

**Pull request merged**